packages/component-library/src/utils/civicFormat.test.js

```diff
@@ -19,6 +19,7 @@ describe('civicFormat', () => {
 
   it('should format dollars correctly', () => {
     expect(civicFormat.dollars(12)).to.eql('$12');
-    expect(civicFormat.dollars(230.5)).to.eql('$230.5');
+    expect(civicFormat.dollars(230.5)).to.eql('$231');
+    expect(civicFormat.dollars(7300000)).to.eql('$7.3 million');
```

Comment:
The removed test made me think about how that's not how money is presented. It would be nice to have a test case for `230.5 -> $230.50`, but no pressure to add that as part of this.